rst_to_md/translator.py

```diff
@@ -263,7 +283,7 @@ def fix_crs(text):
 
         class ListItemContext(Context):
             def finalize(self):
-                front = ['- {}'.format(fix_crs(l)) for l in self.body[:1]]
+                front = ['* {}'.format(fix_crs(l)) for l in self.body[:1]]
```

Comment:
It seems like you made this change (which is fine, as far as I can tell) but didn't update the tests. The tests all expect `-` for bullet lists, but this is generating `*`.